src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -2499,7 +2507,7 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
 
     agent_infra_pool_profile = OpenShiftManagedClusterAgentPoolProfile(
         name='infra',  # Must be 12 chars or less before ACS RP adds to it
-        count=int(2),
+        count=int(3),
```

Comment:
Does this change the default node pool count from 2 to 3? That seems like a significant change (not related to this PR). Maybe it should be mentioned separately in the release notes.

Reply:
Yes, we agreed on this to switch from 2 to 3